src/Microsoft.DotNet.Interactive.SqlServer/MsSqlKernel.cs

```diff
@@ -72,6 +72,7 @@ value switch
                 {
                     string s => $"N{s.AsSingleQuotedString()}",
                     char c => $"N{c.ToString().AsSingleQuotedString()}",
+                    Guid g => $"N{g.ToString().AsSingleQuotedString()}",
```

Comment:
Add test to https://github.com/dotnet/interactive/blob/main/src/Microsoft.DotNet.Interactive.SqlServer.Tests/MsSqlConnectionTests.cs#L300

Reply:
Done.